### DISCUSSION TITLE
Add a history of coordinates to Locality records

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5864] [Feature Requests]
>We are in the process of trying to repatriate data from collaborative georeferencing efforts into our Specify database. In many cases, the localities in question already have older geocoordinates recorded. 

> Ideally, a system could be developed to handle geocoordinates more like taxonomic determinations. In this way, geocoordinate history could be retained in Specify, and a 'Current' checkbox could be used to denote the accepted coordinates.

**Requested By:** Garth at The University of Michigan

**Comment by maxpatiiuk** [comment]
Is the audit log not sufficient for this?
They would like something similar to determinations?

Please also make them aware of the consequences of this (harder to do wb uploads, no nested -to-many support, and etc)

**Comment by g1rly-c0d3r** [comment]
Linked in https://github.com/specify/specify7/issues/8206
